sycl/test/warnings/warnings.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clangxx -Wall -Wpessimizing-move -Wunused-variable -Wmismatched-tags -Wunneeded-internal-declaration -Werror -Wno-unknown-cuda-version -fsycl %s -o %t.out
+// RUN: %clangxx -fsycl --no-system-header-prefix=CL/ -Wall -Wextra -Wno-ignored-attributes -Wno-deprecated-declarations -Wpessimizing-move -Wunused-variable -Wmismatched-tags -Wunneeded-internal-declaration -Werror -Wno-unknown-cuda-version %s -o %t.out
```

Comment:
Minor note: `--no-system-header-prefix=CL/` will most likely enable this check in non-SYCL headers e.g. OpenCL/L0.

I would also add `-c` option to save on linking time.

Reply:
I will do it by separate PR.

Reply:
If you want to save compile time, wouldn't "-fsyntax-only" be even better than "-c" ?